components/schema-migrator/migrations/director/20220622093251_add-runtime-type-labels.up.sql

```diff
@@ -0,0 +1,9 @@
+BEGIN;
+
+INSERT INTO labels(id, tenant_id, app_id, runtime_id, key, value, runtime_context_id, version, app_template_id)
+SELECT uuid_generate_v4() as id, NULL::uuid as tenant_id, NULL::uuid as app_id, t.runtime_id, 'runtimeType' as key, '"kyma"'::jsonb as value, NULL::uuid as runtime_context_id, 0 as version, NULL::uuid as app_template_id
```

Comment:
I left it null because in the labels table the only `key` for which the `runtime_id` and `tenant_id` columns are not null is `scenarios`. For all other keys where the `runtime_id` is not null we do not have any `tenant_id`